spec/factories/player_profiles.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :player_profile do
+    username "MyString"
+    school "MyString"
+    height "MyString"
+    weight "MyString"
+    grade_level 1
+    gpa 1.5
+    guardian_phone "MyString"
+    user nil
```

Comment:
lets make sure this gets changed to mirror the other profiles associations. @brandonrandall